Cargo.lock

```diff
@@ -1026,7 +1026,7 @@ dependencies = [
  "rbpf",
  "serde",
  "serde_json",
- "serial_test",
+ "serial_test 0.5.1",
```

Comment:
Changes in a `Cargo.lock` file appear even on `main` branch just by running `cargo build`. https://github.com/containers/youki/pull/734

Reply:
Yep, I'll revert my change here. :)